test/boost/cql_query_test.cc

```diff
@@ -4313,3 +4315,198 @@ SEASTAR_TEST_CASE(test_rf_expand) {
         });
     });
 }
+
+/*!
+ * \brief this helper function changes all white space into a single space for
+ * a more robust testing.
+ */
+std::string normalize_white_space(const std::string& str) {
+  return std::regex_replace(" " + str + " ", std::regex("\\s+"), " ");
+}
+
+SEASTAR_TEST_CASE(test_describe_simple_schema) {
+    return do_with_cql_env_thread([] (cql_test_env& e) {
+        std::unordered_map<std::string, std::string> cql_create_tables {
+            {"cf", "CREATE TABLE ks.cf (\n"
+            "    pk blob,\n"
+            "    col1 blob,\n"
+            "    col2 blob,\n"
+            "    PRIMARY KEY (pk)\n"
+            ") WITH bloom_filter_fp_chance = 0.01\n"
+            "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+            "    AND comment = ''\n"
+            "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+            "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+            "    AND crc_check_chance = 1\n"
+            "    AND dclocal_read_repair_chance = 0.1\n"
+            "    AND default_time_to_live = 0\n"
+            "    AND gc_grace_seconds = 864000\n"
+            "    AND max_index_interval = 2048\n"
+            "    AND memtable_flush_period_in_ms = 0\n"
+            "    AND min_index_interval = 128\n"
+            "    AND read_repair_chance = 0\n"
+            "    AND speculative_retry = '99.0PERCENTILE';\n"},
+            {"cf1", "CREATE TABLE ks.cf1 (\n"
+                    "    pk blob,\n"
+                    "    ck blob,\n"
+                    "    col1 blob,\n"
+                    "    col2 blob,\n"
+                    "    PRIMARY KEY (pk, ck)\n"
+                    ") WITH CLUSTERING ORDER BY (ck ASC)\n"
+                    "    AND bloom_filter_fp_chance = 0.01\n"
+                    "    AND caching = {'keys': 'ALL','rows_per_partition': 'ALL'}\n"
+                    "    AND comment = ''\n"
+                    "    AND compaction = {'class': 'SizeTieredCompactionStrategy'}\n"
+                    "    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}\n"
+                    "    AND crc_check_chance = 1\n"
+                    "    AND dclocal_read_repair_chance = 0.1\n"
+                    "    AND default_time_to_live = 0\n"
+                    "    AND gc_grace_seconds = 864000\n"
+                    "    AND max_index_interval = 2048\n"
+                    "    AND memtable_flush_period_in_ms = 0\n"
+                    "    AND min_index_interval = 128\n"
+                    "    AND read_repair_chance = 0\n"
+                    "    AND speculative_retry = '99.0PERCENTILE';\n"
+            },
+            {"CF2", "CREATE TABLE ks.\"CF2\" (\n"
+                    "    pk blob,\n"
```

Comment:
It seems in none of these tests you tried a quoted partition-key column (only clustering key), and neither did you try quoted regular column names.